### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD023.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#28] [General]
# https://blog.3place.cn/3place/fsd/FSD023.html

https://blog.3place.cn/3place/fsd/FSD023.html

**Comment by picklecai** [comment]
我多么难过！
墙壁的柱子
包围住我！
圆形屋顶
压住我！——闷气！

当你难过时，感觉什么都在和你作对。
